<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="imgLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>441</width>
     <height>491</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="processedLabel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>441</width>
     <height>491</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>901</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="loadButton">
      <property name="text">
       <string>Load Image</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Save Image</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>60</y>
     <width>181</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="enc1">
      <property name="text">
       <string>Encryption 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="enc2">
      <property name="text">
       <string>Encryption 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="enc3">
      <property name="text">
       <string>Encryption 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="enc4">
      <property name="text">
       <string>Encryption 4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="enc5">
      <property name="text">
       <string>Encryption 5</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>929</x>
     <y>247</y>
     <width>181</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="dec1">
      <property name="text">
       <string>Decryption 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="dec2">
      <property name="text">
       <string>Decryption 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="dec3">
      <property name="text">
       <string>Decryption 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="dec4">
      <property name="text">
       <string>Decryption 4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="dec5">
      <property name="text">
       <string>Decryption 5</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
